src/app/(dashboard)/repositories/[repositoryId]/page.tsx

```diff
@@ -191,14 +194,35 @@ export default function TeamDetailsPage({ params }: PageProps) {
                     ></CalculationIntervalChart>
                   </div>
                 </div>
-
                 <div className="pt-4 grid gap-4 grid-cols-1 md:grid-cols-2 lg:grid-cols-3">
                   <div className="col-span-3 md:col-span-3">
-                    <CalculationResult
-                      userCredDtoList={userCredDtos
-                        .filter((user) => user.type === "USER")
-                        .sort((a, b) => b.totalCred - a.totalCred)}
-                    ></CalculationResult>
+                    <Card className="col-span-2">
+                      <Tabs defaultValue="graph" className="w-full">
+                        <CardHeader>
+                          <CardTitle className="grid grid-cols-2">
+                            <div>Contributors</div>
+                            <TabsList className="grid grid-cols-2 justify-end">
+                              <TabsTrigger value="graph">Graph</TabsTrigger>
+                              <TabsTrigger value="table">Table</TabsTrigger>
+                            </TabsList>
+                          </CardTitle>
+                        </CardHeader>
+                        <CardContent className="pl-2">
+                          <TabsContent value="graph">
+                            <CalculationResult
+                              userCredDtoList={userCredDtos
+                                .filter((user) => user.type === "USER")
+                                .sort((a, b) => b.totalCred - a.totalCred)}
+                            ></CalculationResult>
+                          </TabsContent>
+                          <TabsContent value="table">
+                            <TeamContributionTable
+                              teamId={teamId}
+                            ></TeamContributionTable>
+                          </TabsContent>
+                        </CardContent>
+                      </Tabs>
+                    </Card>
```

Comment:
I'm thinking, If we are now using this in more than once place we should wrapt this whole thing in a comp, Don't need todo it as part of this PR